buildSrc/src/main/groovy/nva.data.report.api.rootplugin.gradle

```diff
@@ -29,3 +29,7 @@ tasks.named('check') {
     dependsOn tasks.named('testCodeCoverageReport', JacocoReport)
     dependsOn tasks.named('integrationTestCodeCoverageReport', JacocoReport)
 }
+
+configurations {
+    compile.exclude module: 'testing-utils'
+}
```

Comment:
With any luck, this is not necessary — it should exclude the testing-utils module from jars if erroneously included, but from what I can see it is excluded from the build lambda anyway